.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -8,7 +8,6 @@ Relates OR Closes #0000
 
 ### Checklist
 - [ ] Added [CHANGELOG](https://github.com/hashicorp/vault-action/blob/master/CHANGELOG.md) entry (only for user-facing changes)
-- [ ] Did not commit changes to `dist/index.js` (This is only done for releases by vault-action maintainers)
```

Comment:
We actually do need to commit the distribution files so that each PR's e2e tests run against the changes.